docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -728,6 +804,9 @@ intentionally use resources without patches and variables to focus on the simple
                 apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
                 kind: VSphereMachineTemplate
                 name: windows-vsphere-template
+          # This will create a health check for each deployment created with the "windows-worker" MachineDeploymentClass
```

Comment:
```suggestion
          # This will create a MachineHealthCheck for each deployment created with the "windows-worker" MachineDeploymentClass
```
nit